dsfs/testdata/complete/template.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
Is there anything that is testing the output of Render, a template that has been processed and had values rendered into it? The name "complete" implies to me that that's what this should be.

Reply:
very good point. In this setting dataset has little-to-no knowledge of what to actually _do_ with the information stored in viz, so we can't test that here. I'm thinking we should rename "complete" to "all_fields", which is what I meant by "complete", and is less ambiguous.